config/terraform.go

```diff
@@ -239,14 +239,15 @@ func (c *TerraformConfig) Validate() error {
 	}
 
 	for k := range c.Backend {
-		if k != "consul" {
+		if k != "consul" && k != "local" {
```

Comment:
This is super small thing and not really much of an issue, particularly with only 2 possibilities, but I find that once you start having more test cases like this it reads better as a switch statement. 
```
switch k {
case "consul", "local": // good
default:
    fmt.Errorf(...)
}

Reply:
Thanks for raising this! I debated going with the switch-case statement for awhile but opted for the if-statement. We'll want to support remote backend (tfc/e) in the near future... I switched to the switch-statement so we can be set-up for success :)